adventure-crew-game/Assets/Scripts/Combat/CombatStatsSO.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "AData", menuName = "ScriptableObjects/CombatStats", order = 1)]
+public class CombatStatsSO : ScriptableObject
+{
+    public int HP;
+    public int Damage;
+    public int Agility;
+    public float Range;
```

Comment:
If we already have the Stats why not make this CombatStats SO have a Stats property? You can serialize structs to be fillable in the inspector and everything